chapter_appendix_math/geometry-linear-algebric-ops_vn.md

```diff
@@ -184,26 +187,27 @@ $$
 to highlight the fact that exchanging the order of the vectors will yield the same answer.
 -->
 
-*dịch đoạn phía trên*
+để nhấn mạnh rằng phép đổi chỗ hai vector sẽ cho kết quả như nhau.
```

Comment:
"phép đổi chỗ" --> mình nghĩ dịch thoáng như sau

để nhấn mạnh rằng trật tự của hai vector trong phép toán là không ảnh hưởng kết quả